test/extensions/filters/http/cache/http_cache_test.cc

```diff
@@ -20,14 +22,15 @@ namespace HttpFilters {
 namespace Cache {
 namespace {
 
+using Seconds = std::chrono::seconds;
+
 struct LookupRequestTestCase {
   std::string test_name, request_cache_control, response_cache_control;
-  SystemTime request_time, response_time;
+  SystemTime request_time, response_date;
```

Comment:
I renamed the `response_time` value to `response_date` to avoid confusion with the `response_time` header/metadata.